community/kernel/src/test/java/org/neo4j/kernel/impl/index/schema/LayoutTestUtil.java

```diff
@@ -86,26 +88,59 @@ void copyValue( VALUE value, VALUE intoValue )
             {
                 Value value;
                 if ( fractionDuplicates > 0 && !uniqueValues.isEmpty() &&
-                        random.nextFloat() < fractionDuplicates )
+                     random.nextFloat() < fractionDuplicates )
                 {
                     value = existingNonUniqueValue( random );
                 }
                 else
                 {
-                    value = newUniqueValue( random, uniqueCompareValues, uniqueValues );
+                    value = newUniqueValue( RandomValues.create( random.random(), getConfiguration() ),
+                            uniqueCompareValues, uniqueValues );
                 }
 
                 return add( currentEntityId++, value );
             }
 
+            private RandomValues.Configuration getConfiguration()
+            {
+                return new RandomValues.Configuration()
+                {
+                    private Randoms.Configuration conf = random.configuration();
```

Comment:
Do we really need `Randoms.Configuration` now that we have `RandomValues.Configuration`?